catroidTest/src/org/catrobat/catroid/uitest/content/brick/PhiroSeekbarColorBrickTest.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2015 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uitest.content.brick;
+
+import android.widget.EditText;
+import android.widget.TextView;
+
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.Script;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.content.StartScript;
+import org.catrobat.catroid.content.bricks.PhiroRGBLightBrick;
+import org.catrobat.catroid.ui.ScriptActivity;
+import org.catrobat.catroid.uitest.util.BaseActivityInstrumentationTestCase;
+import org.catrobat.catroid.uitest.util.UiTestUtils;
+
+public class PhiroSeekbarColorBrickTest extends BaseActivityInstrumentationTestCase<ScriptActivity> {
+	private static final int SET_RED_COLOR_INITIALLY = 0;
+	private static final int SET_GREEN_COLOR_INITIALLY = 10;
+	private static final int SET_BLUE_COLOR_INITIALLY = 15;
+
+	private Project project;
+	private PhiroRGBLightBrick colorBrick;
+
+	public PhiroSeekbarColorBrickTest() {
+		super(ScriptActivity.class);
+	}
+
+	@Override
+	public void setUp() throws Exception {
+		// normally super.setUp should be called first
+		// but kept the test failing due to view is null
+		// when starting in ScriptActivity
+		createProject();
+		super.setUp();
+	}
+
+	public void testRedSeekBar() {
+		//click on the RED editText
+		solo.clickOnView(solo.getView(R.id.brick_phiro_rgb_led_action_red_edit_text));
+		solo.sleep(100);
+		solo.clickOnView(solo.getView(R.id.color_rgb_seekbar_red));
+		solo.sleep(100);
+
+		//check if RED was updated
+		solo.clickOnView(solo.getView(R.id.rgb_red_value));
+		solo.sleep(100);
+		assertEquals("Text not updated within FormulaEditor", 127.0,
+				Double.parseDouble(((EditText) solo.getView(R.id.formula_editor_edit_field)).getText().toString()
+						.replace(',', '.')));
```

Comment:
Hmm, interessting. There is a leading space in the FE edit field. So comparing strings was not possible.

However, perform an [Extract Method](http://refactoring.com/catalog/extractMethod.html) refactoring with the last parameter if there isn't an util method already.